redbot/cogs/mutes/mutes.py

```diff
@@ -321,22 +320,32 @@ async def _auto_channel_unmute_user_multi(
                 self._auto_channel_unmute_user(guild.get_channel(channel), mute_data, False)
             )
         results = await asyncio.gather(*tasks)
+        unmuted_channels = [guild.get_channel(c) for c in channels.keys()]
+        for result in results:
+            if not result:
+                continue
+            _mmeber, channel, reason = result
+            unmuted_channels.pop(channel)
+        modlog_reason = _("Automatic unmute") + "\n".join(
+            [c.mention for c in unmuted_channels if c is not None]
+        )
```

Comment:
I would want to propose alternative formatting for this (you can write the code differently, just wanted to present what I mean by using code):
```suggestion
        modlog_reason = _("Automatic unmute")
        channel_list = humanize_list(
            [c.mention for c in unmuted_channels if c is not None]
        )
        if channel_list:
            modlog_reason += _("\nUnmuted in channels: ") + channel_list
```